metals/src/main/scala/scala/meta/internal/metals/Configs.scala

```diff
@@ -37,6 +37,11 @@ object Configs {
           new FileSystemWatcher(
             Either.forLeft(s"$root/project/build.properties")
           ),
+          new FileSystemWatcher(Either.forLeft(s"$root/**/BUILD")),
+          new FileSystemWatcher(Either.forLeft(s"$root/**/BUILD.bazel")),
+          new FileSystemWatcher(Either.forLeft(s"$root/WORKSPACE")),
+          new FileSystemWatcher(Either.forLeft(s"$root/WORKSPACE.bazel")),
+          new FileSystemWatcher(Either.forLeft(s"$root/**/*.bzl")),
```

Comment:
Ah, we have to register these settings so we make clients to fire `textDocuments/didChange`? nice catch :+1:

Reply:
Actually its `workspace/didChangeWatchedFiles`, ,but it works just as good